### DISCUSSION TITLE
Sliding stops working after opening scaled version of slider

### DISCUSSION THREAD

**Discussion by CodingNinja619** [nolimits4web/swiper#4216] [Q&A] [answered]
I have a slider, and when I click on one of the images it contains, the bigger slider containing the same images is created and displayed, and all other elements are hidden.<br>

But if I click exactly on the first image of the first slider slide a little bit through the bigger slider and then close the bigger slider, the smaller slider stops working: sliding doesn't work. <br>

I noticed that if I disable autoplay property of the first slider, everything works just fine.<br>

jsfiddle of the case when autoplay is enabled (not working): https://jsfiddle.net/seLn6c3w/<br>

jsfiddle of the case when autoplay is disabled (working): https://jsfiddle.net/0dkqubc1/28/<br>

How can I make everything work fine even if autoplay property of the first slider is enabled?<br>

**Comment by nolimits4web** [comment]
You need to stop autoplay on small slider when you hide it, and start autoplay when it becomes again visible

**Reply by CodingNinja619** [reply]
Thank you for your reply, but that didn't work. Maybe I'm doing something wrong:
https://jsfiddle.net/gjyx9hbp/1/

**Reply by CodingNinja619** [reply]
Can you please post a jsfiddle link where your solution would work? I must solve that problem. Thank you in advance

**Comment by CodingNinja619** [comment, marked as answer]
If somebody will ever encounter the same problem as I did, here's the solution.
In album_images_slider click handler (see jsfiddle link) I wrote these lines of code:
```
album_images_slider.slideNext(0, false);
album_images_slider.slidePrev(0, false);
```

This helped. Everyrhing works just fine now
